### PR TITLE
[15.0][FIX] base_tier_validation: Change web_ribbon text to title to make it translatable

### PR DESCRIPTION
**Pull request by victoralmau**
Change web_ribbon text to title to make it translatable

@Tecnativa

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]

File: base_tier_validation/i18n/es.po [hunk 1]
```diff
@@ -621,8 +626,9 @@ msgstr ""
 
 #. module: base_tier_validation
 #: model_terms:ir.ui.view,arch_db:base_tier_validation.res_config_settings_view_form_budget
+#, fuzzy
```
Comment (pedrobaeza) [inline base_tier_validation/i18n/es.po]: Please don't keep fuzzy things.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
It's better to not mix translation work with other things. Keep the PR with the minimum XML change

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**